File: common/src/main/java/com/eliasnvx/krylix/Krylix.java
package com.eliasnvx.krylix;

import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;

public class Krylix {
    public static final String MOD_ID = "krylix";
    public static final String MOD_NAME = "Krylix";

    public static final Logger LOGGER = LogManager.getLogger(MOD_NAME);

    public static void init() {
        LOGGER.info("Initializing " + MOD_NAME);
    }
}

File: common/src/main/kotlin/com/eliasnvx/krylix/Krylix.kt
package com.eliasnvx.krylix

import org.apache.logging.log4j.LogManager
import org.apache.logging.log4j.Logger

/**
 * Главный объект мода Krylix
 * Kill feed мод в стиле CS:GO/Valorant для Minecraft
 */
object Krylix {
    const val MOD_ID = "krylix"
    const val MOD_NAME = "Krylix"

    val LOGGER: Logger = LogManager.getLogger(MOD_NAME)

    /**
     * Инициализация общей (platform-agnostic) части мода, вызывается из KrylixForge.init
     */
    fun init() {
        LOGGER.info("Initializing $MOD_NAME")
    }
}
